src/main/java/ui/components/Notification.java

```diff
@@ -4,25 +4,24 @@
 
 public class Notification {
 
-    public static final int DEFAULT_NOTIFICATION_PANE_VISIBLE_PERIOD = 10;
+    public static final int DEFAULT_NOTIFICATION_PANE_VISIBLE_PERIOD = 30;
 
-    public enum NotificationType { NOACTION, ACTIONONBUTTON, ACTIONONTIMEOUT, ACTIONONBUTTONANDTIMEOUT }
+    @SuppressWarnings("SpellCheckingInspection")
```

Comment:
This is an IntelliJ thing, right? I'm not sure if we want to have it in source code, since it's not IDE-agnostic, and we're going to need it everywhere. Maybe it would be better to have some kind of project-wide configuration to disable spell check? Or let developers do it for themselves.